starter-core/src/test/groovy/io/micronaut/starter/feature/graalvm/GraalNativeImageSpec.groovy

```diff
@@ -95,32 +96,20 @@ class GraalNativeImageSpec extends BeanContextSpec implements CommandOutputFixtu
 
     void 'graalvm feature not supported for Groovy and maven'() {
         when:
-        String template = pom.template(ApplicationType.DEFAULT, buildProject(), getFeatures(["graalvm"], Language.GROOVY), []).render().toString()
+        pom.template(ApplicationType.DEFAULT, buildProject(), getFeatures(["graalvm"], Language.GROOVY), []).render().toString()
 
         then:
-        !template.contains("""
-    <dependency>
-      <groupId>org.graalvm.nativeimage</groupId>
-      <artifactId>svm</artifactId>
-      <scope>provided</scope>
-    </dependency>
-""")
-        !template.contains("""
-    <dependency>
-      <groupId>io.micronaut</groupId>
-      <artifactId>micronaut-graal</artifactId>
-      <scope>provided</scope>
-    </dependency>
-""")
+        def e = thrown(IllegalArgumentException)
+        e.message == 'GraalVM is not supported in Groovy applications'
     }
 
     @Unroll
     void 'deploy.sh script is created for a function with feature graalvm for language: #language'(Language language) {
         when:
         def output = generate(
-                ApplicationType.FUNCTION,
-                new Options(language, TestFramework.JUNIT, BuildTool.MAVEN),
-                ['graalvm', 'aws-lambda-custom-runtime']
+            ApplicationType.FUNCTION,
+            new Options(language, TestFramework.JUNIT, BuildTool.MAVEN, JdkVersion.JDK_11),
```

Comment:
11 is the default. why did you set explicitely Jdk 11 ? Just to be convey that it uses 11?